quadlets/cfe18.container

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=CFE_18 container
+Requires=cfe18mariadb.service
+
+[Container]
+Image=ghcr.io/teragrep/cfe_18:2.1.1
+Volume=/opt/teragrep/cfe_18/app:/etc/tomcat/Catalina/localhost/cfe_18:Z
+PublishPort=8080:8080
+# Static IP for cfe_18
+IP=10.88.0.3
```

Comment:
Similar to what @kortemik posted in https://github.com/teragrep/cfe_18/pull/97#discussion_r1930544443

Where is this sourced from?